src/clr-angular/wizard/_wizard.clarity.scss

```diff
@@ -48,6 +48,9 @@
       display: flex;
       flex-direction: row;
       width: 100%;
+      // align focus style across modal title
```

Comment:
This was a weird rendering bug where the line height was not high enough the focus would render incorrectly on the title element.